cgmanifest.json

```diff
@@ -52,7 +52,7 @@
     },
     {
       "Component": {
-        "Type": "other",
+        "Type": "linux",
```

Comment:
This change alone doesn't fix the issue. The "linux" type cgmanifest has a different JSON schema and info required than that of "other".

Is cgmanifest picking this up correctly? Can you check the docs to make sure we are following the right JSON format?
https://docs.opensource.microsoft.com/tools/cg/features/cgmanifest/

Reply:
See https://dev.azure.com/azure-sdk/public/_componentGovernance/132714?_a=alerts&typeId=3940835&alerts-view-option=active

![image](https://user-images.githubusercontent.com/6527137/132422438-f0d9ce5f-e754-4476-850e-3f89328c0371.png)